d/drupal-flood_control/drupal-flood_control_ubi_8.5.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package       : flood_control
+# Version       : 2.1.0,2.2.2
+# Source repo   : https://git.drupalcode.org/project/flood_control.git
+# Tested on     : UBI 8.5
+# Language      : PHP
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vathsala . <vaths367@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+
+PACKAGE_NAME=flood_control
+PACKAGE_URL=https://git.drupalcode.org/project/flood_control.git
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-2.1.0}
+yum install -y git curl php php-curl php-dom php-mbstring php-json php-gd php-pecl-zip
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && php composer-setup.php --install-dir=/bin --filename=composer
+HOME_DIR=$(pwd)
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    exit 1
+fi
+cd "$HOME_DIR"/$PACKAGE_NAME || exit
+git checkout "$PACKAGE_VERSION"
+# Install symfony/error-handler on compatible PHP versions to avoid a deprecation warning of the old DebugClassLoader and ErrorHandler classes
+composer require --no-update --dev symfony/error-handler "^4.4 || ^5.0"
+composer require --dev phpunit/phpunit --with-all-dependencies ^4
+
+#Tests not available(Test N/A).
```

Comment:
There are no tests available for 2.1.0, but there are [some functional tests available](https://git.drupalcode.org/project/flood_control/-/tree/2.2.2/tests/src/Functional) for the higher version mentioned in the metadata of the build script - 2.2.2

@vathsalashetty25 can you check this once? @seth-priya I'm guessing a configured Drupal webserver will be required for running these testcases